Sources/FluentKit/Query/QueryBuilder.swift

```diff
@@ -41,438 +41,18 @@ public final class QueryBuilder<Model>
-    public func join<ForeignAlias, Local, Value>(
-        _ foreignAlias: ForeignAlias.Type,
-        on filter: JoinFilter<ForeignAlias.Model, Local, Value>,
-        method: DatabaseQuery.Join.Method = .inner
-    ) -> Self
-        where ForeignAlias: ModelAlias, Local: FluentKit.Model
-    {
-        return self.join(
-            ForeignAlias.Model.self, filter.foreign,
-            to: Local.self, filter.local,
-            alias: ForeignAlias.alias
-        )
-    }
-    
-    @discardableResult
-    private func join<Foreign, Local>(
-        _ foreign: Foreign.Type,
-        _ foreignField: String,
-        to local: Local.Type,
-        _ localField: String,
-        method: DatabaseQuery.Join.Method = .inner,
-        alias schemaAlias: String? = nil
-    ) -> Self
-        where Foreign: FluentKit.Model, Local: FluentKit.Model
-    {
-        self.joinedModels.append(.init(model: Foreign(), alias: schemaAlias))
-        self.query.joins.append(.join(
-            schema: .schema(name: Foreign.schema, alias: schemaAlias),
-            foreign: .field(
-                path: [foreignField],
-                schema: schemaAlias ?? Foreign.schema,
-                alias: nil
-            ),
-            local: .field(
-                path: [localField],
-                schema: Local.schema,
-                alias: nil
-            ),
-            method: method
-        ))
-        return self
-    }
-
-    // MARK: Filter
-
-    @discardableResult
-    public func filter(_ filter: ModelValueFilter<Model>) -> Self {
-        return self.filter(
-            .field(path: filter.path, schema: Model.schema, alias: nil),
-            filter.method,
-            filter.value
-        )
-    }
-
-    @discardableResult
-    public func filter(_ filter: ModelFieldFilter<Model, Model>) -> Self {
-        return self.filter(
-            .field(path: filter.lhsPath, schema: Model.schema, alias: nil),
-            filter.method,
-            .field(path: filter.rhsPath, schema: Model.schema, alias: nil)
-        )
-    }
-
-    @discardableResult
-    public func filter<Left, Right>(_ filter: ModelFieldFilter<Left, Right>) -> Self
-        where Left: FluentKit.Model, Right: FluentKit.Model
-    {
-        return self.filter(
-            .field(path: filter.lhsPath, schema: Left.schema, alias: nil),
-            filter.method,
-            .field(path: filter.rhsPath, schema: Right.schema, alias: nil)
-        )
-    }
-
-    @discardableResult
-    public func filter<Joined, Values>(
-        _ field: KeyPath<Joined, Field<Values.Element>>,
-        in values: Values,
-        alias: String? = nil
-    ) -> Self
-        where Joined: FluentKit.Model, Values: Collection, Values.Element: Codable
-    {
-        return self.filter(Joined.self, Joined.key(for: field), in: values, alias: alias)
-    }
-    
-    @discardableResult
-    public func filter<Values>(_ field: KeyPath<Model, Field<Values.Element>>, in values: Values) -> Self
```

Comment:
@tanner0101 was it intentional to remove these `filter(_, in:)` helpers?

Reply:
@namolnad yes that's because the `~~` operators support this + more. Otherwise lots of overloads need to be duplicated. 
```swift
filter(\.$names, in: arrayOfNames)
// can be
filter(\.$names ~~ arrayOfNames)
```

Reply:
Makes sense. Thanks for confirming!